src/cli.h

```diff
@@ -2,6 +2,7 @@
 #define CLI_H
 
 #include <glib.h> /* gboolean and GOptionEntry */
+#include <stdint.h>
```

Comment:
can you add a comment /* unit64_t */ to both?

Reply:
Done